docs/plugins/integrations.asciidoc

```diff
@@ -29,13 +28,24 @@ make it easier to work with Elasticsearch.
   search (facets, etc), along with some Natural Language Processing features
   (ex.: More like this)
 
-
 [float]
 [[data-integrations]]
 === Data import/export and validation
 
-NOTE: Rivers were used to import data from external systems into
-Elasticsearch, but they are no longer supported in Elasticsearch 2.0.
+NOTE: Rivers were used to import data from external systems into Elasticsearch prior to the 2.0 release. Elasticsearch 
+releases 2.0 and later do not support rivers.
+
+[float]
+==== Supported by the community:
```

Comment:
Is it really a river replacement, though? Rivers are their own functionality. While these aren't ES plugins, they are integrations between ES and LS, so I think they fall under the general title.